src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -10824,12 +10821,16 @@ record C(object P)
     public object get_P() => null;
     public object set_Q() => null;
 }";
-            var comp = CreateCompilation(source);
+            var comp = CreateCompilation(source, targetFramework: TargetFramework.StandardLatest);
             comp.VerifyDiagnostics(
                 // (9,17): error CS0082: Type 'C' already reserves a member called 'get_P' with the same parameter types
                 // record C(object P)
                 Diagnostic(ErrorCode.ERR_MemberReserved, "P").WithArguments("get_P", "C").WithLocation(9, 17));
 
+            var expectedClone = comp.Assembly.RuntimeSupportsCovariantReturnsOfClasses
```

Comment:
>comp.Assembly.RuntimeSupportsCovariantReturnsOfClasses [](http://example.com/codeflow?start=32&length=54)

I think we would want to add an assert that this value is true when RuntimeUtilities.IsCoreClrRuntime is true. Same for the other PR that fixes dead code in tests

Reply:
I am suggesting doing this in every test that is supposed to observe the difference based on ```Assembly.RuntimeSupportsCovariantReturnsOfClasses```